versioned/persist/nontx/src/main/java/org/projectnessie/versioned/persist/nontx/NonTransactionalDatabaseAdapter.java

```diff
@@ -853,4 +997,72 @@ protected abstract GlobalStateLogEntry fetchFromGlobalLog(
 
   protected abstract List<GlobalStateLogEntry> fetchPageFromGlobalLog(
       NonTransactionalOperationContext ctx, List<Hash> hashes);
+
+  protected Hash writeRefLogEntry(
```

Comment:
This should live in `ABstractDatabaseAdapter`

Reply:
As mentioned above,
I intentionally didn't added there to avoid having ReflogEntry to adaptertypes.ReflogEntry conversion. If I keep it inside like writeGlobalCommit, I can avoid that extra conversion as I can build adaptertypes.ReflogEntry directly.

So, I am basically following like writeGlobalCommit